tooling/cli/src/mobile/mod.rs

```diff
@@ -178,13 +182,20 @@ fn setup_dev_config(
       let ip = crate::dev::local_ip_address(force_ip_prompt);
       url.set_host(Some(&ip.to_string())).unwrap();
       if let Some(c) = config_extension {
-        let mut c: tauri_utils::config::Config = serde_json::from_str(c)?;
-        c.build.dev_url = dev_url.clone();
-        config_extension.replace(serde_json::to_string(&c).unwrap());
+        if let Some(build) = c
+          .0
+          .as_object_mut()
+          .and_then(|root| root.get_mut("build"))
+          .and_then(|build| build.as_object_mut())
+        {
+          build.insert("devUrl".into(), url.to_string().into());
+        }
       } else {
-        config_extension.replace(format!(r#"{{ "build": {{ "devUrl": "{url}" }} }}"#));
+        config_extension.replace(ConfigValue(
+          serde_json::from_str(&format!(r#"{{ "build": {{ "devUrl": "{url}" }} }}"#)).unwrap(),
+        ));
```

Comment:
would it be faster to use serde_json::json! macro here?

Reply:
oops i checked it and couldn't find how to add dynamic values, turns out it was in the macro docs itself :joy: